tests/validators/test_float.py

```diff
@@ -11,6 +11,7 @@
 
 from ..conftest import Err, PyAndJson, plain_repr
 
+i64_max = 9_223_372_036_854_775_807
```

Comment:
what do you think about expressing this as

```suggestion
i64_max = (2**63)-1
```